spec/rails_helper.rb

```diff
@@ -4,6 +4,9 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'rake/dsl_definition'
+require 'coveralls'
```

Comment:
I put the coveralls code in my spec_helper.rb.

Please move for consistency. :shaved_ice: